R/doeFactorial.R

```diff
@@ -842,6 +862,11 @@ doeFactorial <- function(jaspResults, dataset, options, ...){
     )
   }
 
+  # Replace : by \u00d7
+  for (i in 1:length(anovaFill$Terms)) {
+    anovaFill$Terms[i] <- gsub(":", " \u00d7 ", anovaFill$Terms[i])
+  }
```

Comment:
is this for an interaction symbol? if yes, you can replace this by
```suggestion
  anovaFill$Terms <- jaspBase::gsubInteractionSymbol(anovaFill$Terms)
```
which will transform e.g., `"factor:covariate"` into `"factor ✻ covariate"`